srcts/src/shiny/index.ts

```diff
@@ -54,7 +56,7 @@ interface Shiny {
   renderContent: typeof renderContent;
   renderHtmlAsync: typeof renderHtmlAsync;
   renderHtml: typeof renderHtml;
-  user: string;
+  user?: string;
```

Comment:
I added the `?` here, which reflects how the code actually is written. (The conversion from interface to class flagged this issue.)